IKEA/IKEA/SupportingView/RecentProductSectionView.swift

```diff
@@ -0,0 +1,119 @@
+        collectView.showsHorizontalScrollIndicator = false
+        collectView.register(
+            RecentProductCollectionViewCell.self,
+            forCellWithReuseIdentifier: "RecentProductCollectionViewCell"
+        )
+        collectView.translatesAutoresizingMaskIntoConstraints = false
+
+        return collectView
+    }()
+    
+    // MARK: - Init
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupLayout()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Method
+    func setupLayout() {
+        [
+            titleLabel,
+            deleteLabel,
+            RecentProductCollectView
+        ].forEach{ self.addSubview($0) }
+
+        NSLayoutConstraint.activate([
+            titleLabel.topAnchor.constraint(equalTo: self.topAnchor, constant: 30),
+            titleLabel.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: 30)
+        ])
+
+        NSLayoutConstraint.activate([
+            deleteLabel.topAnchor.constraint(equalTo: self.topAnchor, constant: 30),
+            deleteLabel.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant:  -30)
+        ])
+
+        NSLayoutConstraint.activate([
+            RecentProductCollectView.topAnchor.constraint(equalTo: titleLabel.bottomAnchor, constant: 35),
+            RecentProductCollectView.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: 0),
+            RecentProductCollectView.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant:  0),
+            RecentProductCollectView.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: 0),
+            RecentProductCollectView.heightAnchor.constraint(equalToConstant: 80)
+        ])
+    }
+
+}
+
+// MARK: - CollectionViewDataSource
+extension RecentProductSectionView: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        6
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: RecentProductCollectionViewCell.identifier, for: indexPath) as! RecentProductCollectionViewCell
+        cell.layer.cornerRadius = cell.frame.size.height / 2.0
+        cell.clipsToBounds = true
+        cell.layer.borderWidth = 0.5
+        cell.layer.borderColor = UIColor.systemGray2.cgColor
+        return cell
+    }
+}
+
+// MARK: - CollectionViewDelegateFlowLayout
+
+extension RecentProductSectionView: UICollectionViewDelegateFlowLayout{
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        let cellSize = RecentProductCollectView.frame.width / 5
+        return CGSize(width: cellSize, height: cellSize)
+    }
```

Comment:
```suggestion
      let size = RecentProductCollectView.frame.width / 5
        let cellSize = CGSize(width: size, height: size)
        return cellSize
    }
```
진짜진짜 별거는 아닌데 일단 코드 리뷰할 거리를 만들기 위해서 생각한 대안입니다. 
이런식으로 한번 더 프로퍼티로 담아서 리턴하면 추후에 혹여나 크기 수정할때에 더 용이한 코드가 될 수 있지 않을까..!! 해서 
간언드리옵니다 폐하.

Reply:
음.. 저는 이미 `cellSize`라는 변수에 크기 값을 담았기 때문에 
추후 수정 시에  `cellSize` 변수만 바꾸면 될 것 같다라는 생각이 드는데 
(제가 아직 잘 몰라서..)

`CGSize`를 한번 더 변수로 선언했을 때 어떤 점이 더 편한지 아직은 잘 모르겠는데
혹시 예시를 말씀해 주실 수 있으신가요?🥹🥺